src/main/kotlin/org/elm/lang/core/types/TypeRendering.kt

```diff
@@ -26,7 +26,9 @@ fun TyUnion.renderedText(linkify: Boolean, withModule: Boolean): String {
     val type = when {
         parameters.isEmpty() -> name
         else -> parameters.joinToString(" ", prefix = "$name ") {
-            if (it is TyFunction || it is TyUnion && it.parameters.isNotEmpty()) {
+            if (it is TyFunction
+                    || it is TyUnion && it.parameters.isNotEmpty()
+                    || it.alias?.parameters?.isNotEmpty() == true) {
```

Comment:
minor thing: remove the `== true` part

Reply:
`it.alias?.parameters?.isNotEmpty()` has type `Boolean?`, so the `== true` is required.